### PR TITLE
Bluetooth: Audio: Update BT_BAP_SCAN_DELEGATOR dependency to GATT_DYN…

### LINKED ISSUE #79108: Bluetooth: Audio: Change dependency of BT_BAP_SCAN_DELEGATOR to GATT_DYNAMIC_DB from 'select' to 'depends on'
**Is your enhancement proposal related to a problem? Please describe.**

Currently, in `subsys/bluetooth/audio/Kconfig.bap`, `BT_BAP_SCAN_DELEGATOR` uses `select` to manage its dependency on `GATT_DYNAMIC_DB`. The `select` mechanism can create issues because it forces the selection of certain options without checking if those dependencies are met. This can lead to configurations that are invalid or unexpected.

**Describe the solution you'd like**

I propose changing the dependency management of `BT_BAP_SCAN_DELEGATOR` from `select` to `depends on`. Using `depends on` ensures that the dependency on `GATT_DYNAMIC_DB` is only satisfied if it has already been selected or enabled elsewhere, thus preventing the automatic selection that can lead to invalid configurations. This approach promotes better modularity and clarity in configuration management.

**Describe alternatives you've considered**

One option is to retain the current configuration using `select`, perhaps with enhanced documentation to explain its use and implications. However, this approach does not eliminate the risk of generating invalid configurations.

**Additional context**

This change might introduce a dependency loop, which will need to be carefully investigated and resolved. Ensuring that dependencies are correctly managed without loops is crucial for maintaining a stable and predictable build system. Further testing and validation will be required to ensure that this change does not negatively impact other parts of the system.

### PR DESCRIPTION
**Pull request by babrsn**
…AMIC_DB

Changed dependency of the BT_BAP_SCAN_DELEGATOR to GATT_DYNAMIC_DB from 'select' to 'depends on' and solved all loop dependencies caused by this change.

Fixes #79108

### CODE REVIEW

**Comment by PavelVPV** [comment]
I guess mesh changed by coincidence?

File: tests/bluetooth/tester/overlay-le-audio.conf [hunk 1]
```diff
@@ -41,6 +41,7 @@ CONFIG_BT_ISO_TX_BUF_COUNT=4
 CONFIG_BT_BAP_BROADCAST_ASSISTANT=y
 
 # Broadcast Sink
+CONFIG_BT_GATT_DYNAMIC_DB=y
```
Comment (sjanc) [inline tests/bluetooth/tester/overlay-le-audio.conf:44]: this is already set in prj.conf

**Comment by omkar3141** [comment]
> I guess mesh changed by coincidence?

I believe this is very relevant for mesh since we change services at run-time after provisioning (removal of Mesh Provisioning Service and exposing Mesh Proxy Service). I also compiled 'light' sample on v2.6.0 and I see `CONFIG_BT_GATT_DYNAMIC_DB=y` in generated `.config` file.

**Review by omkar3141 — DISMISSED** [review]
Looks good for mesh.

**Comment by PavelVPV** [comment]
> Looks good for mesh.

Should be removed. See comment above.

File: subsys/bluetooth/audio/Kconfig.bap [hunk 2]
```diff
@@ -166,7 +166,7 @@ config BT_BAP_BROADCAST_SINK
 	bool "Bluetooth Broadcast Sink Audio Support"
 	select BT_ISO_SYNC_RECEIVER
 	select BT_AUDIO_RX
-	select BT_PAC_SNK
+	depends on BT_PAC_SNK
```
Comment (Thalley) [inline subsys/bluetooth/audio/Kconfig.bap:169]: I assume this is due some circular dependency?

File: subsys/bluetooth/mesh/Kconfig [hunk 3]
```diff
@@ -37,7 +37,7 @@ config BT_MESH_GATT_CLIENT
 config BT_MESH_GATT_SERVER
 	bool
 	select BT_MESH_GATT
-	select BT_GATT_DYNAMIC_DB
+	depends on BT_GATT_DYNAMIC_DB
```
Comment (Thalley) [inline subsys/bluetooth/mesh/Kconfig:40]: Does not changing this generate build warnings/errors?

**Review by Thalley — CHANGES_REQUESTED** [review]
Since this will likely break one or more applications (given the changes required to the `.conf` files, I suggest we put an entry for both LE Audio and Mesh in the migration guide for this. 

Ideally the changes to mesh shouldn't be affected by this, but I assume there's some build errors if not changed?

**Comment by babrsn** [comment]
> > > I guess mesh changed by coincidence?
> > 
> > 
> > I believe this is very relevant for mesh since we change services at run-time after provisioning (removal of Mesh Provisioning Service and exposing Mesh Proxy Service).
> > I also compiled 'light' sample on NCSv2.6.0 and I see `CONFIG_BT_GATT_DYNAMIC_DB=y` in generated `.config` file. Same observed for Zephyr 'mesh' sample on corresponding zephyr revision.
> 
> No need for explicit selection of Kconfig options that have been enabled by default for ages:
> 
> https://github.com/zephyrproject-rtos/zephyr/blob/d022d315d2e006ad1eb13f3f72a00a99b4ff7fb0/subsys/bluetooth/mesh/Kconfig#L37-L40

This change is to resolve the dependency loop issue.

File: tests/bluetooth/tester/overlay-le-audio.conf (hunk 1 above)
Comment (babrsn) [reply tests/bluetooth/tester/overlay-le-audio.conf:44]: Ok.

File: subsys/bluetooth/audio/Kconfig.bap (hunk 2 above)
Comment (babrsn) [reply subsys/bluetooth/audio/Kconfig.bap:169]: Yes, that's true.

File: subsys/bluetooth/mesh/Kconfig (hunk 3 above)
Comment (babrsn) [reply subsys/bluetooth/mesh/Kconfig:40]: Yes, it leads to build errors related to the dependency loop, and it is one of the easiest ways to solve it.

**Comment by PavelVPV** [comment]
> > > > I guess mesh changed by coincidence?
> > > 
> > > 
> > > I believe this is very relevant for mesh since we change services at run-time after provisioning (removal of Mesh Provisioning Service and exposing Mesh Proxy Service).
> > > I also compiled 'light' sample on NCSv2.6.0 and I see `CONFIG_BT_GATT_DYNAMIC_DB=y` in generated `.config` file. Same observed for Zephyr 'mesh' sample on corresponding zephyr revision.
> > 
> > 
> > No need for explicit selection of Kconfig options that have been enabled by default for ages:
> > https://github.com/zephyrproject-rtos/zephyr/blob/d022d315d2e006ad1eb13f3f72a00a99b4ff7fb0/subsys/bluetooth/mesh/Kconfig#L37-L40
> 
> This change is to resolve the dependency loop issue.

Could you elaborate more on this? I can't understand what dependency loop happens by reading the description of the issue.

**Comment by babrsn** [comment]
> Since this will likely break one or more applications (given the changes required to the `.conf` files, I suggest we put an entry for both LE Audio and Mesh in the migration guide for this.
> 
> Ideally the changes to mesh shouldn't be affected by this, but I assume there's some build errors if not changed?

I agree that it would be better if the mesh wasn’t affected by this change, but as I mentioned before, it was changed to resolve the dependency loop issue. I think this is the easiest way to do it, otherwise we would need to make more changes to avoid circular dependency.

> > > > > I guess mesh changed by coincidence?
> > > > 
> > > > 
> > > > I believe this is very relevant for mesh since we change services at run-time after provisioning (removal of Mesh Provisioning Service and exposing Mesh Proxy Service).
> > > > I also compiled 'light' sample on NCSv2.6.0 and I see `CONFIG_BT_GATT_DYNAMIC_DB=y` in generated `.config` file. Same observed for Zephyr 'mesh' sample on corresponding zephyr revision.
> > > 
> > > 
> > > No need for explicit selection of Kconfig options that have been enabled by default for ages:
> > > https://github.com/zephyrproject-rtos/zephyr/blob/d022d315d2e006ad1eb13f3f72a00a99b4ff7fb0/subsys/bluetooth/mesh/Kconfig#L37-L40
> > 
> > 
> > This change is to resolve the dependency loop issue.
> 
> Could you elaborate more on this? I can't understand what dependency loop happens by reading the description of the issue.

[report.txt](https://github.com/user-attachments/files/17277400/report.txt)

Ok, to describe it in more detail, assume I just change the dependency of ´BT_BAP_SCAN_DELEGATOR´ from ´select´ to ´depends on´ in ´subsys/bluetooth/audio/Kconfig.bap´ (as intended by this issue). Then, for instance, when I build ´bap_broadcast_sink´, I get the dependency loop error (see the attachment file). To resolve it, the dependency loop must be broken. In my opinion, this is the easiest way to solve it, otherwise we would need to change more files and areas.

**Comment by PavelVPV** [comment]
> [report.txt](https://github.com/user-attachments/files/17277400/report.txt)
> 
> Ok, to describe it in more detail, assume I just change the dependency of ´BT_BAP_SCAN_DELEGATOR´ from ´select´ to ´depends on´ in ´subsys/bluetooth/audio/Kconfig.bap´ (as intended by this issue). Then, for instance, when I build ´bap_broadcast_sink´, I get the dependency loop error (see the attachment file). To resolve it, the dependency loop must be broken. In my opinion, this is the easiest way to solve it, otherwise we would need to change more files and areas.

Why do you want to change `select BT_GATT_DYNAMIC_DB` to `depends on BT_GATT_DYNAMIC_DB` for `BT_BAP_SCAN_DELEGATOR`?

**Comment by Thalley** [comment]
> > [report.txt](https://github.com/user-attachments/files/17277400/report.txt)
> > Ok, to describe it in more detail, assume I just change the dependency of ´BT_BAP_SCAN_DELEGATOR´ from ´select´ to ´depends on´ in ´subsys/bluetooth/audio/Kconfig.bap´ (as intended by this issue). Then, for instance, when I build ´bap_broadcast_sink´, I get the dependency loop error (see the attachment file). To resolve it, the dependency loop must be broken. In my opinion, this is the easiest way to solve it, otherwise we would need to change more files and areas.
> 
> Why do you want to change `select BT_GATT_DYNAMIC_DB` to `depends on BT_GATT_DYNAMIC_DB` for `BT_BAP_SCAN_DELEGATOR`?

Besides being semantically better when dealing with `depend`encies, our Kconfig page also mentions that `depends on` is usually better than `select` for a handful of reason (including these circular dependencies that we have...): https://docs.zephyrproject.org/latest/build/kconfig/tips.html#alternatives-to-select

**Comment by Thalley** [comment]
> > > Why do you want to change `select BT_GATT_DYNAMIC_DB` to `depends on BT_GATT_DYNAMIC_DB` for `BT_BAP_SCAN_DELEGATOR`?
> > 
> > 
> > Besides being semantically better when dealing with `depend`encies, our Kconfig page also mentions that `depends on` is usually better than `select` for a handful of reason (including these circular dependencies that we have...): https://docs.zephyrproject.org/latest/build/kconfig/tips.html#alternatives-to-select
> 
> Since it was working fine before even being semantically not perfect, I'd really like to know what is actually wrong (not worse/better) before approving because now all users will have to explicitly enable `BT_GATT_DYNAMIC_DB` in their project files.

> Since it was working fine before

The fact that LE audio cannot use `depends on` (and thus also no other application) with Mesh enabled is evidence, IMO, that it was not working fine before. The current usage of `select` is effectively forcing others to also using `select` rather than `depends on`. 

@babrsn It would be interesting to see how the changes are if Mesh wasn't changed. You mentioned that the circular dependency could be fixed in another way, without affecting Mesh?

**Comment by PavelVPV** [comment]
> The fact that LE audio cannot use `depends on` (and thus also no other application) with Mesh enabled is evidence, IMO, that it was not working fine before.

From what I see this doesn't depend on whether mesh enabled or not:
> Ok, to describe it in more detail, assume I just change the dependency of ´BT_BAP_SCAN_DELEGATOR´ from ´select´ to ´depends on´ in ´subsys/bluetooth/audio/Kconfig.bap´ (as intended by this issue). Then, for instance, when I build ´bap_broadcast_sink´, I get the dependency loop error (see the attachment file).

This should be explained in the issue more clearly.

**Comment by babrsn** [comment]
> > > > Why do you want to change `select BT_GATT_DYNAMIC_DB` to `depends on BT_GATT_DYNAMIC_DB` for `BT_BAP_SCAN_DELEGATOR`?
> > > 
> > > 
> > > Besides being semantically better when dealing with `depend`encies, our Kconfig page also mentions that `depends on` is usually better than `select` for a handful of reason (including these circular dependencies that we have...): https://docs.zephyrproject.org/latest/build/kconfig/tips.html#alternatives-to-select
> > 
> > 
> > Since it was working fine before even being semantically not perfect, I'd really like to know what is actually wrong (not worse/better) before approving because now all users will have to explicitly enable `BT_GATT_DYNAMIC_DB` in their project files.
> 
> > Since it was working fine before
> 
> The fact that LE audio cannot use `depends on` (and thus also no other application) with Mesh enabled is evidence, IMO, that it was not working fine before. The current usage of `select` is effectively forcing others to also using `select` rather than `depends on`.
> 
> @babrsn It would be interesting to see how the changes are if Mesh wasn't changed. You mentioned that the circular dependency could be fixed in another way, without affecting Mesh?

At the moment, I don’t have a ready-to-submit alternative solution. However, I believe there should be more alternatives for this issue. I think the current solution here is the simplest because I have mostly changed the configs which select `BT_GATT_DYNAMIC_DB`. I suspect that alternative solutions will affect more parameters in more places. If it would be beneficial, I can investigate to see if I can find another good solution that doesn’t affect 'Mesh'. I could do this tomorrow or the day after...

**Comment by Thalley** [comment]
> At the moment, I don’t have a ready-to-submit alternative solution. However, I believe there should be more alternatives for this issue. I think the current solution here is the simplest because I have mostly changed the configs which select `BT_GATT_DYNAMIC_DB`. I suspect that alternative solutions will affect more parameters in more places. If it would be beneficial, I can investigate to see if I can find another good solution that doesn’t affect 'Mesh'. I could do this tomorrow or the day after...

I think it's worth investigating alternative solutions that doesn't affect mesh (and preferably nothing else besides the LE Audio stack). If you do get something else working, please submit that as a separate commit, either in this PR or another for comparison

**Comment by PavelVPV** [comment]
Perhaps you want something like this: https://github.com/zephyrproject-rtos/zephyr/commit/39eb100948c8380164634bed0a8dfccba31a4c01 ?

All audio and mesh upstream samples compile with this change.

But as you see now you (and all who use audio) need explicitly enable BT_GATT_DYNAMIC_DB and some other Bluetooth options.

If you want to proceed with this, I'd recommend go through audio Kconfigs and replace "select BT_" to "depends on BT_" for all upper Bluetooth Kconfig options.

**Comment by omkar3141** [comment]
> If you want to proceed with this, I'd recommend go through audio Kconfigs and replace "select BT_" to "depends on BT_" for all upper Bluetooth Kconfig options.

It is not a bad idea, though it will make all "prj.conf" look quite big and littered with many KConfig options.

Then I think, may be it is worth to not introduce this change everywhere and do it only at a place where it actually creates a problem (like proposed in this PR). This could be middle ground even though it is not so pure and pious.

**Comment by Thalley** [comment]
> Perhaps you want something like this: [39eb100](https://github.com/zephyrproject-rtos/zephyr/commit/39eb100948c8380164634bed0a8dfccba31a4c01) ?
> 
> All audio and mesh upstream samples compile with this change.
> 
> But as you see now you (and all who use audio) need explicitly enable BT_GATT_DYNAMIC_DB and some other Bluetooth options.
> 
> If you want to proceed with this, I'd recommend go through audio Kconfigs and replace "select BT_" to "depends on BT_" for all upper Bluetooth Kconfig options.

That's closer to the result I had expected from fixing the issue as well. Ideally we want to replace all `select` with `depends on` in LE audio in the future

> It is not a bad idea, though it will make all "prj.conf" look quite big and littered with many KConfig options.
> 
> Then I think, may be it is worth to not introduce this change everywhere and do it only at a place where it actually creates a problem (like proposed in this PR). This could be middle ground even though it is not so pure and pious.

I think this is a subjective matter. 
Besides `depends on` being suggested by Zephyr over `select`, forcing applications to enable things explicitly does indeed make the `.conf` files larger, but also clearer in terms of what is enabled. 

For example enabling `CONFIG_FOO` should in many cases perhaps not enable 10s of other Kconfigs what the knowledge of the user. See https://docs.zephyrproject.org/latest/build/kconfig/tips.html#select-pitfalls for reasons why `select` should not be overused.

**Comment by babrsn** [comment]
> > Thanks. Although it involves a considerable number of changes in the config symbols within the Kconfig files
> 
> As mentioned earlier, these changes are likely ones we want in the future anyways. We should be able to replace all `select` by `depends on` in LE Audio.

Indeed

**Review by Thalley — DISMISSED** [review]

File: samples/bluetooth/cap_acceptor/prj.conf [hunk 4]
```diff
@@ -1,6 +1,9 @@
 CONFIG_BT=y
 CONFIG_LOG=y
 CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline samples/bluetooth/cap_acceptor/prj.conf:4]: Why do we need the observer here?

File: samples/bluetooth/bap_broadcast_assistant/prj.conf [hunk 5]
```diff
@@ -1,6 +1,7 @@
 CONFIG_BT=y
 CONFIG_LOG=y
 CONFIG_BT_CENTRAL=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline samples/bluetooth/bap_broadcast_assistant/prj.conf:4]: The observer is implied for centrals
```suggestion
CONFIG_BT_CENTRAL=y
```

File: samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf [hunk 6]
```diff
@@ -13,6 +13,9 @@ CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
 CONFIG_LTO=y
 
 CONFIG_BT=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
```
Comment (Thalley) [inline samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf:18]: Ditto, observer implied by central
```suggestion
CONFIG_BT_PERIPHERAL=y
CONFIG_BT_CENTRAL=y
```

File: samples/bluetooth/hci_uart/overlay-all-bt_ll_sw_split.conf [hunk 7]
```diff
@@ -10,6 +10,9 @@ CONFIG_BT_EXT_ADV=y
 CONFIG_BT_PER_ADV=y
 CONFIG_BT_PER_ADV_SYNC=y
 CONFIG_BT_PER_ADV_SYNC_MAX=2
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
```
Comment (Thalley) [inline samples/bluetooth/hci_uart/overlay-all-bt_ll_sw_split.conf:15]:
```suggestion
CONFIG_BT_PERIPHERAL=y
CONFIG_BT_CENTRAL=y
```

File: samples/bluetooth/hci_uart_3wire/overlay-all-bt_ll_sw_split.conf [hunk 8]
```diff
@@ -10,6 +10,9 @@ CONFIG_BT_EXT_ADV=y
 CONFIG_BT_PER_ADV=y
 CONFIG_BT_PER_ADV_SYNC=y
 CONFIG_BT_PER_ADV_SYNC_MAX=2
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
```
Comment (Thalley) [inline samples/bluetooth/hci_uart_3wire/overlay-all-bt_ll_sw_split.conf:15]:
```suggestion
CONFIG_BT_PERIPHERAL=y
CONFIG_BT_CENTRAL=y
```

File: tests/bluetooth/init/prj_ctlr_5_x_dbg.conf [hunk 9]
```diff
@@ -1,4 +1,5 @@
 CONFIG_BT=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bluetooth/init/prj_ctlr_5_x_dbg.conf:2]:
```suggestion
```

Central is enabled

File: tests/bluetooth/shell/audio.conf [hunk 10]
```diff
@@ -13,6 +13,9 @@ CONFIG_BT_CENTRAL=y
 CONFIG_BT_ISO_CENTRAL=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_PRIVACY=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bluetooth/shell/audio.conf:16]:
```suggestion
```
Ditto

File: tests/bluetooth/shell/log.conf [hunk 11]
```diff
@@ -7,6 +7,7 @@ CONFIG_BT=y
 CONFIG_BT_CENTRAL=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_PRIVACY=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bluetooth/shell/log.conf:10]:
```suggestion
```

File: tests/bluetooth/shell/prj.conf [hunk 12]
```diff
@@ -8,6 +8,7 @@ CONFIG_LOG=y
 CONFIG_BT_CENTRAL=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_PRIVACY=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bluetooth/shell/prj.conf:11]:
```suggestion
```

File: tests/bluetooth/tester/overlay-le-audio.conf [hunk 13]
```diff
@@ -1,5 +1,9 @@
 CONFIG_BT_AUDIO=y
 CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bluetooth/tester/overlay-le-audio.conf:4]:
```suggestion
CONFIG_BT_CENTRAL=y
```

File: tests/bsim/bluetooth/ll/bis/prj.conf [hunk 14]
```diff
@@ -1,4 +1,5 @@
 CONFIG_BT=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bsim/bluetooth/ll/bis/prj.conf:2]:
```suggestion
```

File: tests/bsim/bluetooth/ll/bis/prj_vs_dp.conf [hunk 15]
```diff
@@ -2,6 +2,7 @@ CONFIG_BT=y
 CONFIG_BT_DEVICE_NAME="ISO"
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_CENTRAL=y
+CONFIG_BT_OBSERVER=y
```
Comment (Thalley) [inline tests/bsim/bluetooth/ll/bis/prj_vs_dp.conf:5]:
```suggestion
```

File: tests/bsim/bluetooth/ll/cis/sysbuild/hci_ipc/nrf5340_cpunet_iso_acl_group-bt_ll_sw_split.conf [hunk 16]
```diff
@@ -10,6 +10,9 @@ CONFIG_HEAP_MEM_POOL_SIZE=8192
 CONFIG_CBPRINTF_REDUCED_INTEGRAL=y
 
 CONFIG_BT=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
```
Comment (Thalley) [inline tests/bsim/bluetooth/ll/cis/sysbuild/hci_ipc/nrf5340_cpunet_iso_acl_group-bt_ll_sw_split.conf:15]:
```suggestion
CONFIG_BT_PERIPHERAL=y
CONFIG_BT_CENTRAL=y
```

**Review by Thalley — CHANGES_REQUESTED** [review]

File: samples/bluetooth/cap_acceptor/prj.conf (hunk 4 above)
Comment (babrsn) [reply samples/bluetooth/cap_acceptor/prj.conf:4]: Just for the BSIM test with `CONFIG_SAMPLE_BROADCAST=y`, alternatively `select BT_OBSERVER` can be added to the `config SAMPLE_BROADCAST` in `cap_acceptor/Kconfig`.

Comment (Thalley) [reply samples/bluetooth/cap_acceptor/prj.conf:4]: Ah, that makes sense. 

It should however be moved to where the Kconfig options for broadcast is enabled: https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/bluetooth/cap_acceptor/Kconfig#L21-L32

Comment (babrsn) [reply samples/bluetooth/cap_acceptor/prj.conf:4]: Yes, I mentioned the same place in my comment above! I’ll move it there.

File: samples/bluetooth/bap_broadcast_assistant/prj.conf (hunk 5 above)
Comment (babrsn) [reply samples/bluetooth/bap_broadcast_assistant/prj.conf:4]: That's true. I'll correct it.

File: samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf (hunk 6 above)
Comment (babrsn) [reply samples/bluetooth/hci_ipc/nrf5340_cpunet_iso-bt_ll_sw_split.conf:18]: Ok.

File: samples/bluetooth/hci_uart_3wire/overlay-all-bt_ll_sw_split.conf (hunk 8 above)
Comment (babrsn) [reply samples/bluetooth/hci_uart_3wire/overlay-all-bt_ll_sw_split.conf:15]: OK

File: tests/bluetooth/init/prj_ctlr_5_x_dbg.conf (hunk 9 above)
Comment (babrsn) [reply tests/bluetooth/init/prj_ctlr_5_x_dbg.conf:2]: OK

File: tests/bluetooth/shell/audio.conf (hunk 10 above)
Comment (babrsn) [reply tests/bluetooth/shell/audio.conf:16]: OK

File: tests/bluetooth/shell/log.conf (hunk 11 above)
Comment (babrsn) [reply tests/bluetooth/shell/log.conf:10]: OK

File: tests/bluetooth/shell/prj.conf (hunk 12 above)
Comment (babrsn) [reply tests/bluetooth/shell/prj.conf:11]: OK

File: tests/bluetooth/tester/overlay-le-audio.conf (hunk 13 above)
Comment (babrsn) [reply tests/bluetooth/tester/overlay-le-audio.conf:4]: OK

File: tests/bsim/bluetooth/ll/bis/prj.conf (hunk 14 above)
Comment (babrsn) [reply tests/bsim/bluetooth/ll/bis/prj.conf:2]: OK.

File: tests/bsim/bluetooth/ll/bis/prj_vs_dp.conf (hunk 15 above)
Comment (babrsn) [reply tests/bsim/bluetooth/ll/bis/prj_vs_dp.conf:5]: OK

File: tests/bsim/bluetooth/ll/cis/sysbuild/hci_ipc/nrf5340_cpunet_iso_acl_group-bt_ll_sw_split.conf (hunk 16 above)
Comment (babrsn) [reply tests/bsim/bluetooth/ll/cis/sysbuild/hci_ipc/nrf5340_cpunet_iso_acl_group-bt_ll_sw_split.conf:15]: Ok.

**Comment by Thalley** [comment]
@babrsn needs a rebase for the CCID changes I believe :)

**Review by Thalley — APPROVED** [review]

**Review by niym-ot — APPROVED** [review]

**Pull request merged**